@here/harp-mapview/lib/text/TextElementBuilder.ts

```diff
@@ -83,8 +99,11 @@ export class TextElementBuilder {
      */
     constructor(
         private readonly m_env: MapEnv | Env,
-        private readonly m_styleCache: TileTextStyleCache
-    ) {}
+        private readonly m_styleCache: TileTextStyleCache,
+        private readonly m_baseRenderOrder: number
```

Comment:
I saw that later in the tests, you pass a lot of zeros for the baseRenderOrder, maybe you can default to 0 instead

Reply:
I rather pass the extra zeros explicitely in the test. Having a default value has the risk that someone forgets to pass the datasourceOrder.